### DISCUSSION TITLE
Hello from Indonesia, and fellow learners around the World!

### DISCUSSION THREAD

**Discussion by weisurya** [OpenMined/courses#103] [Introduce Yourself]
Hi everyone, I'm Surya from Indonesia. My background is on the technical side for a fraud risk prevention team in the payment gateway company. I started to be interested in data privacy and private AI field due to fact that my day-to-day work is kind of in that area as well. I hope we could enjoy this learning journey together and gain some insight to make a positive impact on ourselves and our society.

If you have a LinkedIn account, let's connect! Here's mine: [https://www.linkedin.com/in/weisurya/](https://www.linkedin.com/in/weisurya/)

**Comment by iamtrask** [comment]
Hi Surya - welcome! Your work does seem quite relevant! What kind of role do you have - would love to hear about where you think it's likely to be adopted earlier - and also things that might be farther down the road!

**Comment by Physicist91** [comment]
Hey there! Nice to see someone else from Indonesia too 👍

**Comment by yemikifouly** [comment]
Hello @weisurya and @Physicist91!!! Welcome to the discussion board to both of you! 🙂
